docs/templates/hooks/more-head.tpl

```diff
@@ -0,0 +1,68 @@
+<!-- Stork search styling, scripts added in after-body.tpl -->
+<link rel="stylesheet" href="https://files.stork-search.net/releases/v1.5.0/basic.css" />
+<style>
```

Comment:
Maybe more suiting to have this as a snippet variable instead?

Reply:
I've moved it into being a snippet. I felt it was easier to manage when handling supporting multiple layouts by more easily switch between which script collection to use.